getting-started-with-reactions/README.md

```diff
@@ -0,0 +1,32 @@
+# Getting Started with Message Reactions
+
+`getting-started-with-reactions` is a basic 1:1 chat application that features a message input field
+and a list of messages that pile up on the screen as you send them. In addition to this basic
+functionality, the app contains 6 built-in message reactions that you can add to the messages by
+long-tapping on a message and selecting emojis from the bottom drawer that slides up from the bottom
+edge of the screen when triggered.
+
+<img src="../assets/getting-started-with-reactions.png" alt="Getting Started app for Android" style="width:300px"/> 
+
+## Prerequisites
+
+This application uses [PubNub Kotlin SDK](https://github.com/pubnub/kotlin) (>= 7.0.0) for chat
+components and [Jetpack Compose](https://developer.android.com/jetpack/compose) as the UI Toolkit.
+
+To use the app, you need:
+
+* [Android Studio](https://developer.android.com/studio/preview) (>= Bumblebee 2021.1.1)
+* PubNub account on [Admin Portal](https://dashboard.pubnub.com/)
+
+## Usage
+
+Read the [Getting Started](https://www.pubnub.com/docs/chat/components/android/get-started-android)
```

Comment:
Maybe we should point to the Message Reactions doc here (as well)? It's because we will describe in this doc how to enable reactions based on that app?